test/e2e/samsahai/ctrl.go

```diff
@@ -1004,6 +1004,80 @@ var _ = Describe("[e2e] Main controller", func() {
 
 	}, 30)
 
+	It("should successfully delete active environment", func(done Done) {
+		defer close(done)
+		setupSamsahai(true)
+		ctx := context.TODO()
+
+		By("Creating Config")
+		config := mockConfig
+		Expect(client.Create(ctx, &config)).To(BeNil())
+
+		By("Creating Team")
+		team := mockTeam
+		team.Status.Namespace.Active = atvNamespace
+		team.Status.ActivePromotedBy = "user"
+		Expect(client.Create(ctx, &team)).To(BeNil())
+
+		By("Verifying namespace and config have been created")
+		err = wait.PollImmediate(verifyTime1s, verifyNSCreatedTimeout, func() (ok bool, err error) {
+			namespace := corev1.Namespace{}
+			if err := client.Get(ctx, types.NamespacedName{Name: stgNamespace}, &namespace); err != nil {
+				return false, nil
+			}
+
+			config := s2hv1beta1.Config{}
+			err = client.Get(ctx, types.NamespacedName{Name: team.Name}, &config)
+			if err != nil {
+				return false, nil
+			}
+
+			return true, nil
+		})
+		Expect(err).NotTo(HaveOccurred(), "Verify namespace and config error")
+
+		By("Active Environment should not be deleted")
```

Comment:
there is no active namespace at the beginning, is there?

Reply:
no, already added at step creating team
https://github.com/agoda-com/samsahai/blob/c1eaf24f52728b30d8e49f56d3bd1d9126bf47b1/test/e2e/samsahai/ctrl.go#L1018

Reply:
I meant the active namespace has not been created, has it?